### PR TITLE
Add subsections on "migration of the objects"

### PR DESCRIPTION
**Pull request by CrystalBallBe**
I added subsections:
 - un-migratable objects
   - finalized markets
   - dispute windows

This is the last part of the Fork section.
@MicahZoltu Could you review this pull request?

### CODE REVIEW

**Comment by CrystalBallBe** [comment]
Demo: https://crystalballbe.github.io/version2-english/9-fork/4-migration-of-the-objects.html#un-migratable-objects

File: collections/_version2-english/9-fork/4-migration-of-the-objects.md [hunk 1]
```diff
@@ -138,10 +140,13 @@ All [REP]({{glossary_reputation_token}}) in the [parent universe]({{glossary_par
 
 If you are the first person to migrate REP to a [child universe]({{glossary_child_universe}}), child universe creation is triggered and the forking market is also copied into it. In this case, you have to pay the [transaction fee]({{glossary_transaction_fee}}) not only for migrating your REP but also for creating the child universe and copying the forking market.
 
----
-
-*The rest is in progress.*
-
 # Un-migratable objects
 ## Finalized Markets
+The finalized markets and its attached objects ([share]({{glossary_share}}), [unfilled order]({{glossary_unfilled_order}}), and [creator fee]({{glossary_creator_fee}})) can not be migrated. However, you can still trade and [settle]({{glossary_settlement}}) your [shares]({{glossary_share}}) on the markets during/after a [fork]({{glossary_fork}}).
```
Comment (MicahZoltu) [inline collections/_version2-english/9-fork/4-migration-of-the-objects.md]:
```suggestion
Finalized markets and their attached objects ([shares]({{glossary_share}}), [unfilled orders]({{glossary_unfilled_order}}), and [creator fee]({{glossary_creator_fee}})) can not be migrated. However, you can still trade and [settle]({{glossary_settlement}}) your [shares]({{glossary_share}}) on the markets during/after a [fork]({{glossary_fork}}).
```

File: collections/_version2-english/9-fork/4-migration-of-the-objects.md [hunk 2]
```diff
@@ -138,10 +140,13 @@ All [REP]({{glossary_reputation_token}}) in the [parent universe]({{glossary_par
...
+
 ## Dispute Windows
+[Dispute Windows]({{glossary_dispute_window}}) and its attached objects ([reporting fee pool]({{glossary_reporting_fee_pool}}), [reporting fee]({{glossary_reporting_fee}}), [validity bond]({{glossary_validity_bond}}), and [participation token]({{glossary_participation_token}}),) can not be migrated.
```
Comment (MicahZoltu) [inline collections/_version2-english/9-fork/4-migration-of-the-objects.md]:
```suggestion
[Dispute Windows]({{glossary_dispute_window}}) and their attached objects ([reporting fee pool]({{glossary_reporting_fee_pool}}), [reporting fees]({{glossary_reporting_fee}}), [validity bonds]({{glossary_validity_bond}}), and [participation tokens]({{glossary_participation_token}})) can not be migrated.
```

File: collections/_version2-english/9-fork/4-migration-of-the-objects.md [hunk 3]
```diff
@@ -138,10 +140,13 @@ All [REP]({{glossary_reputation_token}}) in the [parent universe]({{glossary_par
...
+
+Before and after a [fork]({{glossary_fork}}), there is no change in the features of a dispute window. That means Augur corrects reporting fee and validity bond, and adds those to reporting fee pool during a dispute window. At the end of the dispute window, the pool is paid out to participation token owners in proportion to the amount of participation token.
```
Comment (MicahZoltu) [inline collections/_version2-english/9-fork/4-migration-of-the-objects.md]:
```suggestion
Before and after a [fork]({{glossary_fork}}), there is no change in the features of a dispute window. This means Augur collects reporting fees and validity bonds, and adds those to reporting fee pool during a dispute window. At the end of the dispute window, the pool is paid out to participation token owners in proportion to the amount of participation token.
```

**Review by MicahZoltu — APPROVED** [review]
Just some pluralization stuff, looks good!

Did you verify in the code that dispute windows don't migrate?

**Comment by CrystalBallBe** [comment]
@MicahZoltu 
Thank you for reviewing. 
I didn't verify that.  I don't have time right now, so I'll try to verify that later.

**Comment by MicahZoltu** [comment]
If it is true that dispute windows don't migrate, that is unfortunately a bug/problem.  I'm hoping you are wrong, but I kind of suspect you are right.

**Comment by CrystalBallBe** [comment]
I'm looking into the contracts, and  I think dispute windows do not migrate. And it seems they are created in a child universe when the non-forking market is first migrated to it.

I think 👇 is the line where dispute windows are created.
https://github.com/AugurProject/augur/blob/959cabf1e3079cfdf7e4c351b6a6c9df336c6dea/packages/augur-core/src/contracts/reporting/Market.sol#L426

And now I wonder why you think that is a bug... Ah, you think reporting fees that are pooled in the reporting fee pool should follow their market to a child universe, right?

**Comment by CrystalBallBe** [comment]
>  it seems they are created in a child universe when the non-forking market is first migrated to it.

"a child universe" here👆 is the winning universe. I'm looking into where the dispute windows of losing universes are created in the contracts.

**Comment by CrystalBallBe** [comment]
I looked into the contracts, and I guess  the dispute windows of losing universes are created here👇
https://github.com/AugurProject/augur/blob/959cabf1e3079cfdf7e4c351b6a6c9df336c6dea/packages/augur-core/src/contracts/reporting/Market.sol#L126

So I think:
 - When a losing universe is created, the dispute window is not created yet.
 - When the initial reporting is first done in the losing universe,  the dispute window is created.

**Comment by MicahZoltu** [comment]
The issue is that settlement fees for markets that resolved prior to the fork will go to reporting windows on the parent universe.  If we assume that all honest REP migrates to a child universe during the fork, this means all of those fees will be "stuck" in the parent universe.

**Comment by CrystalBallBe** [comment]
I see. If all REP migrates to a child universe, no one can purchase participation tokens in the parent universe. Then, the fees in the pool in the parent universe will never be distributed. 

That means REP in the parent universe will not lose their value, even after a fork ends. If huge fees should be expected to be pooled in the parent universe, that weakens the motivation to migrate REP.

In order to further the migration, the value of REP in the parent universe should be completely lost after a fork ends. To do so, the fees that are paid in the parent universe should be pooled in the winning universe after finalizing the forking market I think.

**Pull request merged**